components/claim/claimable-rewards.tsx

```diff
@@ -249,16 +255,16 @@ const ClaimableRewards: React.FC<ClaimableRewardsProps> = ({ isAdmin }) => {
                 claimStatus={
                   item.claimed
                     ? "claimed"
-                    : item.buttonDisabled
-                      ? "expired"
+                    : item.reason
```

Comment:
does swapping out `buttonDisabled` for `reason` cause any issues?

Reply:
no, it just makes the button text more descriptive because buttonDisabled can be true for 2 other reasons besides "Claimed"